packages/prosemirror-lwdita/scripts/fix.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
```

Comment:
Nothing should come before the license decl, can you move this down please?

Reply:
Done.